tests/components/signal_messenger/test_notify.py

```diff
@@ -48,82 +47,88 @@ def setUp(self):
             recipients, client
         )
 
-    @requests_mock.Mocker()
-    def test_send_message(self, mock):
+    def test_send_message(self, requests_mock, caplog):
         """Test send message."""
+
+        self.setUp()
```

Comment:
I believe the most "pytest appropriate" way to handle this would be to make SignalNotificationService a pytest fixture, then accepted as an argument to the test method.